manager/controllers/app/config/default_evaluator.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
```

Comment:
I would suggest calling it adminconfig (or something similar) so it's not confused with confused with fybrik/config

Reply:
OK.